pkg/cmd/create/create_devpod.go

```diff
@@ -158,6 +158,20 @@ func NewCmdCreateDevPod(commonOpts *opts.CommonOptions) *cobra.Command {
 	return cmd
 }
 
+// GetGitLabels generate label keys and values for git repository elements (scheme, host, org and repo)
+func GetGitLabels(gitInfo *gits.GitRepository) (string, string, string, string, string, string, string, string) {
+	gitSchemeLabelKey := fmt.Sprintf("%s/scheme", kube.LabelDevPodGitPrefix)
+	gitSchemeLabelValue := gitInfo.Scheme
+	gitHostLabelKey := fmt.Sprintf("%s/host", kube.LabelDevPodGitPrefix)
+	gitHostLabelValue := gitInfo.Host
```

Comment:
for good measure we might want to wrap each of the values into a `ToValidNameTruncated`

Reply:
ummm .. We don't have problem with scheme or host, but org or repo .. we can clash with another repo if we cut by length on this variable.

Maybe, cut scheme and host, but return error if the length of org & repo variables exceed the length ?

Reply:
> but org or repo .. we can clash with another repo if we cut by length on this variable.

wonder what the concrete consequence, in this case, would be. Any idea?

>  Maybe, cut scheme and host, but return error if the length of org & repo variables exceed the length ?

Works for me. I guess now that the labels are split the scenario becomes quite unlikely really.